Hi,

Previous issue: #2348

The above had a fix but the fix only seems to apply when using "source" to specify an authorised keys file.

When just using the name to set a key - it does not run restorecon in set_auth_key at https://github.com/saltstack/salt/blob/develop/salt/modules/ssh.py#L473 if it generates a new file.

Hopefully an easy one :)

Thanks,

Jason
